control_room/main.py

```diff
@@ -77,8 +90,21 @@ def initialize_exe_modules(mod_cfgs: dict) -> list[ModuleConnection]:
     """
     connections = []
 
-    # Python modules
+    if "modules" not in mod_cfgs:
+        logger.info("No exe modules to initialize.")
+        return connections
+
+    logger.debug(f"Exe module configurations found: {mod_cfgs['modules'].keys()}")
+
+    # Exe modules
     for module_name, module_cfg in mod_cfgs["modules"].items():
+        required_keys = ["path", "ip", "port", "pcomms"]
```

Comment:
The `pcomms` would not necessarily come from the config. The usual case is that the module, once started, is requested to send available pcomms, using the GET_PCOMMS. I think we need to remove the pcomms from the `required_keys` list